modules/ROOT/pages/dw-operators.adoc

```diff
@@ -386,39 +386,58 @@ xref:dataweave-variables.adoc#example_local_variable[Examples: Local DataWeave V
 
 DataWeave supports the `update` operator, which enables you to update specified fields of a data structure with new values.
 
-_Introduced in DataWeave 2.3.0. Supported by Mule 4.3 and later._ +
+_Introduced in DataWeave 2.3.0. Supported by Mule 4.3 and later._
 
-The following example shows use of the update operator, assume that you receive the next object as input:
-[source,json,linenums]
+Versions prior to Mule 4.3 require the following syntax to increase the `age` value in `myInput` by one without recreating the entire object:
```

Comment:
possible rephrasing 

```suggestion
To increase the `age` value in `myInput` by one without recreating the entire object, versions prior to Mule 4.3 require the following syntax:
```

Reply:
We want to emphasize the version here by putting it in front because we are contrasting that with the previous entry that involves what you had to do prior to v4.3.